content/learn/beginner/datatypes-1.md

```diff
@@ -0,0 +1,131 @@
+numbers together, but you cannot add two strings together (at least not in the way you might expect).
+
+## Strings
+
+A string is a sequence of characters. You can think of it as a word or a sentence. In the bot's template actions, you
+can create a string by enclosing some text in double quotes (`"`). For instance, `"Hello, world!"` is a string. We call
+those *quoted strings*.
+
+Now, here we might run into a problem quite quickly: what if we want to include a double quote in our string? We can't
+just write `"Peter said "Hello, world!""`, as the bot would interpret this in a nonsensical way. To solve this, we
+must escape the double quote by adding a backslash (`\`) in front of it. This tells the bot that the double quote is
+part of the string and not the end of it. In other words, `"Peter said \"Hello, world!\""` would yield the expected
+result.
+
+To insert a newline (you would press `Enter` on your keyboard), you can use the escape sequence `\n`. Consider the
+string `"Hello\nWorld!"`, which would result in the following output:
+
+```txt
+Hello
+World!
+```
+
+For a full list of escape sequences, you can refer to the [Go documentation](https://golang.org/ref/spec#Rune_literals).
+Please note that not all escape sequences are supported by Discord.
+
+### Raw String Literals
+
+It should become relatively clear that a lot of new lines and other special characters can make a quoted string quite
+hard to read. To make this easier, you can use backticks (`` ` ``) to create a *raw string literal*. This enables you to
+*literally* press enter to insert a new line in your string, without having to use the `\n` escape sequence, like so:
+
+```txt
+`This is my
+cool multiline string!
+
+Look at all the new lines!`
+```
+
+To insert a backtick in a raw string literal, however, you must use its ASCII code, which is `\x60`. Escape sequences as
+in quoted strings are not supported.
+
+## Numbers
+
+Numeric values can be represented in two ways, using integers (whole numbers) and floating-point numbers (numbers with a
+decimal point). In the bot's template actions, you can create an integer by simply writing a number, such as `5`. For
+floating-point numbers, you can add a decimal point, like so: `5.0`.
+
+### Integers
+
+In the bot's template actions, integers are represented as 64-bit signed integers. This means that you can store numbers
+from `-9223372036854775808` to `9223372036854775807`. If you try to store a number outside this range, the bot will
+return an error.
+
+There are several ways to define an integer, specifically:
+
+1. As a base-10 number, such as `42`. This will mean what you think, the number forty-two.
+2. As a base-16 number, such as `0x2A`. This is the [hexadecimal representation][hex] of the number forty-two.
+3. As a base-8 number, such as `052`. This is the [octal representation][oct] of the number forty-two.
```

Comment:
I would suggest using the `0o` prefix instead, which Go supports and is less likely to cause confusion. (Many newer languages only support `0o` octal literals for this reason.)